posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: "Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reece-nana
+author_github: https://github.com/reece-nana
+seo-title: Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+blog_description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta brings you the hot off the press [Jakarta EE 10 Core Profile](https://jakarta.ee/specifications/coreprofile/10/). Jakarta EE 10 is the first release that provides some updates to the Jakarta EE specifications since Java EE 8 in 2017. While Jakarta EE 10 release is around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  In this beta release, the features that make up the new Jakarta EE 10 Core Profile are made available: 
```

Comment:
```suggestion
Open Liberty 22.0.0.9-beta brings you the hot off the press link:https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]. Jakarta EE 10 is the first release that provides some updates to the Jakarta EE specifications since Java EE 8 in 2017. While Jakarta EE 10 release is around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  In this beta release, the features that make up the new Jakarta EE 10 Core Profile are made available: 
```